packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketSelectedOffer.tsx

```diff
@@ -29,36 +33,40 @@ export const CoinmarketSelectedOffer = () => {
     const paymentMethod = getPaymentMethod(context.selectedQuote, context);
 
     return (
-        <Wrapper>
-            {isCoinmarketBuyContext(context) && (
-                <CoinmarketOfferBuy
-                    account={account}
-                    selectedQuote={context.selectedQuote}
-                    providers={providers}
-                    type={context.type}
-                    quoteAmounts={quoteAmounts}
-                    {...paymentMethod}
-                />
-            )}
-            {isCoinmarketSellContext(context) && (
-                <CoinmarketOfferSell
-                    account={account}
-                    selectedQuote={context.selectedQuote}
-                    providers={providers}
-                    type={context.type}
-                    quoteAmounts={quoteAmounts}
-                    {...paymentMethod}
-                />
-            )}
-            {isCoinmarketExchangeContext(context) && (
-                <CoinmarketOfferExchange
-                    account={account}
-                    selectedQuote={context.selectedQuote}
-                    providers={providers}
-                    type={context.type}
-                    quoteAmounts={quoteAmounts}
-                />
-            )}
-        </Wrapper>
+        <Column gap={spacings.md}>
+            {!device?.connected && <ConnectDeviceGenericPromo />}
```

Comment:
For now, you can buy when device is disconnected

![Screenshot 2024-12-27 at 12 23 36](https://github.com/user-attachments/assets/1cd888dd-4e84-4ad4-a09b-2cfe98894252)

Reply:
Hmm but only if you take best offer, in case you go to all offers, you cannot. cc @adderpositive @FreeWall . Probably bug?
![Screenshot 2024-12-27 at 12 23 59](https://github.com/user-attachments/assets/25722f8f-b654-4e1a-884d-41a3b5a9e168)

Reply:
@TomasBoda It would be great if you could fix it. Please, I appreciate it. 🙏 Also, please check the states of the buttons in the **swap** and **sell** sections when you randomly connect or disconnect the device.